tests/build_tests/CMakeLists.txt

```diff
@@ -69,6 +69,12 @@ ENDIF()
 
 FOREACH(_step_full ${_steps})
   GET_FILENAME_COMPONENT(_step ${_step_full} NAME)
+  # Skip real-value only PETSc tests:
+  IF (DEAL_II_PETSC_WITH_COMPLEX)
+    IF ("${_step}" MATCHES "^(step-55|step-40|step-18|step-17)$")
+      CONTINUE ()
```

Comment:
where do you want to add it? Testes are declared below `ADD_TEST(NAME ${_test}`. I don't know how @tamiko parses normal `xyz.output` files for these `with_xyz` options, but I believe this is irrelevant here, because all tests are added based on directory structure in `${_binary_dir}/examples/`, there are no`.output` files here. These are build-only tests.